src/components/app/Home.css

```diff
@@ -121,6 +121,40 @@
   font-size: 16px;
   text-align: center;
   cursor: pointer;
+  background-color: #fff;
+}
+
+.homeSectionButtonPressed {
+  padding: 0.5em;
+  margin: 15px 0;
+  display: block;
+  border: 2px solid var(--blue-50);
+  text-decoration: none;
+  transition: background-color 300ms;
+  font-weight: bold;
+  font-size: 13px;
+  text-align: center;
+  cursor: pointer;
+  background-color: var(--blue-50);
+  color: #fff;
+
+}
+
+.homeSectionActionButtons {
+  display: flex;
+  justify-content: space-between;
+}
+
+.homeSectionLoadFromUrl {
+  width: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.homeSectionActionButtons .homeSectionButton , .homeSectionLoadFromUrl .homeSectionButton {
+  font-size: 13px;
```

Comment:
yes i know , but i use this due to the note of greg about make the font size and padding of the three action buttons smaller than others + i use 13 specifically to make some margin between buttons since font size above 13 remove the margin.

Reply:
@julienw  did you mean at `.homeSectionActionButtons`? 
the second selector is for Load button in case of load from URL

Reply:
Yes I understand we need 2 selectors, but then each selector can be simpler with only 1 class name.

Reply:
Yes but only for the font-size. But don't worry too much about this, it's not a big deal.